rfc/3/index.md

```diff
@@ -0,0 +1,158 @@
+# RFC-3: more dimensions for thee
+
+Remove restrictions on the number of dimensions stored in OME-NGFF arrays.
+
+## Status
+
+Draft (D3).
+
+| Name      | GitHub Handle | Institution | Date       | Status                                 |
+|-----------|---------------|-------------|------------|----------------------------------------|
+| Davis Bennett | [d-v-b](https://github.com/d-v-b) | N/A | 2024-05-01 | [Implementing][implementation] |
+| Juan Nunez-Iglesias | [jni](https://github.com/jni) | Monash University | 2024-05-21 | Author |
+| Talley Lambert | [tlambert03](https://github.com/tlambert03) | Harvard Medical School | 2024-05-21 | [Endorse](https://github.com/ome/ngff/pull/239#issuecomment-2122795327) | -->
+| Norman Rzepka | [normanrz](https://github.com/normanrz) | Scalable Minds | 2024-05-21 | [Endorse](https://github.com/ome/ngff/pull/239#issue-2308436425) | -->
+| Davis Bennett | [d-v-b](https://github.com/d-v-b) |  | 2024-05-21 | [Endorse](https://github.com/ome/ngff/pull/239#issue-2308436425) | -->
+| Doug Shepherd | [dpshepherd](https://github.com/dpshepherd) | Arizona State University | 2024-05-22 | [Endorse](https://github.com/ome/ngff/pull/239#issue-2308436425) | -->
+| John Bogovic | [bogovicj](https://github.com/bogovicj) | HHMI Janelia Research Campus | 2024-05-22 | [Endorse](https://github.com/ome/ngff/pull/239#issue-2308436425) | -->
+| Eric Perlman | [perlman](https://github.com/perlman) |  | 2024-05-22 | [Endorse](https://github.com/ome/ngff/pull/239#issue-2308436425) | -->
+| Lachlan Deakin | [LDeakin](https://github.com/LDeakin) | Australian National University | 2024-05-22 | [Endorse](https://github.com/ome/ngff/pull/239#issue-2308436425) | -->
+
+## Overview
+
+OME-NGFF version 0.4 places severe restrictions on the number, names, and types
+of axes that are allowed in the axes metadata. This has had the effect of
+limiting the datasets in proprietary formats that *can* be meaningfully
+converted to NGFF. It has also prevented some novel datasets from being written
+in NGFF format. This RFC only removes restrictions from the specification. An
+important consequence is that all valid NGFF datasets would remain valid after
+this change.
```

Comment:
I had this formatted as a suggestion but the formatting got in the way. I think it might help to put some text in there like this: 

"These restrictions hinder converting some datasets in proprietary formats to NGFF. The axis restrictions have also prevented some novel datasets from being written in the NGFF format. Clearly the data model articulated in version 0.4 of OME-NGFF is not inclusive enough for these datasets. So this RFC proposes removing these axis restrictions, thereby making OME-NGFF more open to the diversity of bioimaging datasets.  Because this RFC only removes restrictions from the specification, all valid NGFF datasets remain valid after this change."

basically, the 2-5D restriction is a barrier to researchers who have data that doesn't fit into that model. so we remove the restriction, and make the ome-ngff model more expressive, so that those researchers can use the format.